tests/.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+C:37:"PHPUnit\Runner\DefaultTestResultCache":1072:{a:2:{s:7:"defects";a:5:{s:60:"Spaze\PHPStan\Rules\Disallowed\NamespaceUsagesTest::testRule";i:3;s:56:"Spaze\PHPStan\Rules\Disallowed\StaticCallsTest::testRule";i:3;s:56:"Spaze\PHPStan\Rules\Disallowed\MethodCallsTest::testRule";i:3;s:59:"Spaze\PHPStan\Rules\Disallowed\ConstantUsagesTest::testRule";i:3;s:64:"Spaze\PHPStan\Rules\Disallowed\ClassConstantUsagesTest::testRule";i:3;}s:5:"times";a:9:{s:71:"Spaze\PHPStan\Rules\Disallowed\ClassConstantInvalidUsagesTest::testRule";d:0.19;s:64:"Spaze\PHPStan\Rules\Disallowed\ClassConstantUsagesTest::testRule";d:0.051;s:59:"Spaze\PHPStan\Rules\Disallowed\ConstantUsagesTest::testRule";d:0.005;s:54:"Spaze\PHPStan\Rules\Disallowed\EvalCallsTest::testRule";d:0.021;s:58:"Spaze\PHPStan\Rules\Disallowed\FunctionCallsTest::testRule";d:0.003;s:56:"Spaze\PHPStan\Rules\Disallowed\MethodCallsTest::testRule";d:0.035;s:53:"Spaze\PHPStan\Rules\Disallowed\NewCallsTest::testRule";d:0.017;s:56:"Spaze\PHPStan\Rules\Disallowed\StaticCallsTest::testRule";d:0.02;s:60:"Spaze\PHPStan\Rules\Disallowed\NamespaceUsagesTest::testRule";d:0.202;}}}
```

Comment:
Please remove the file.

Reply:
🤦 done

Reply:
The file was still there but this time I could remove it myself.